word_parser.rb

```diff
@@ -0,0 +1,52 @@
+module WordParser
+  def self.read_file(file_path)
+    File.open(file_path) { |f| f.read }
+  end
+
+  def self.to_arrayed_words(string)
+    string.gsub!(/\s\-\s/, ' ')
+    string.downcase.split(/\s/)
+  end
+
+  def self.clean_word(word)
+    word.gsub!(/[^a-zA-Z\-']/, '')
+    strip_single_quotes(word)
+  end
+
+  def self.clean_array(array)
+    array.map { |word| clean_word(word) }
+    remove_empty_strings(array)
+  end
+
+  def self.counted(array)
+    word_hash = {}
```

Comment:
Very nice...was able to eliminate the ternary operator by doing this and that's generally a good thing to do.